turbopack/crates/turbo-tasks-macros/src/derive/mod.rs

```diff
@@ -31,34 +31,49 @@ impl From<&[Attribute]> for FieldAttributes {
 
         for attr in attrs {
             if !attr
-                .path
+                .path()
                 .get_ident()
                 .map(|ident| *ident == "turbo_tasks")
                 .unwrap_or_default()
             {
                 continue;
             }
-            if let Ok(Meta::List(MetaList { nested, .. })) = attr
-                .parse_meta()
-                .map_err(|err| err.span().unwrap().error(err.to_string()).emit())
+            let nested = match attr.parse_args_with(Punctuated::<Meta, Token![,]>::parse_terminated)
             {
-                for meta in nested {
-                    if let NestedMeta::Meta(Meta::Path(path)) = &meta {
-                        match path.get_ident().map(|ident| ident.to_string()).as_deref() {
-                            Some("trace_ignore") => result.trace_ignore = true,
-                            Some("debug_ignore") => result.debug_ignore = true,
-                            _ => path
+                Ok(punctuated) => punctuated,
+                Err(e) => {
+                    attr.meta
+                        .span()
+                        .unwrap()
+                        .error(format!(
+                            "expected `trace_ignore` or `debug_ignore`, got: {}",
+                            e
+                        ))
```

Comment:
Nit:

```suggestion
                        .error(format!(
                            "expected `trace_ignore` or `debug_ignore`, got: {e}",
                        ))
```

A lot of older code doesn't do this as the syntax was only added in 2022 for 1.58.0: https://blog.rust-lang.org/2022/01/13/Rust-1.58.0/#captured-identifiers-in-format-strings